suspect/processing/channel_combination.py

```diff
@@ -3,14 +3,19 @@
 
 def svd_weighting(data):
     p, _, v = numpy.linalg.svd(data, full_matrices=False)
+    
     channel_weights = p[:, 0].conjugate()
+    # normalize so they sum to 1
+    channel_weights = channel_weights/numpy.sum(numpy.abs(channel_weights))
 
     # try some basic phase correction
     # in our truncation of the SVD to rank 1, we know that v[0] is our FID
     # use the first point of it to phase the signal
     phase_shift = numpy.angle(v[0, 0])
+    
+    channel_weights_phase_corrected = channel_weights * numpy.exp(-1j * phase_shift)        
 
-    return channel_weights * numpy.exp(-1j * phase_shift) / numpy.sum(numpy.abs(channel_weights))
+    return channel_weights_phase_corrected, channel_weights, phase_shift
```

Comment:
Also there is no reason that the "unphased" weights from the wref are any better as an option for the phase of the suppressed signal.